tests/MatrixDotNet.PerformanceTesting/MathExtension/AddTest.cs

```diff
@@ -18,9 +18,9 @@ public void Setup()
         }
 
         [Benchmark]
-        public Matrix<double> AddUnsafe()
+        public Matrix<double> AddOptimizationUnsafe()
         {
-            return Matrix<double>.Plus(_matrix1,_matrix2);
+            return _matrix1 + _matrix2;
```

Comment:
We have this code on lines 27-30 in this file

Reply:
ok I remove it